Formula/orientdb.rb

```diff
@@ -39,4 +39,10 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    # quick check if file is there
```

Comment:
Sorry, I'll be clearer. 😺 I'm not sure how to start/stop a database from Ruby (using Ruby system commands?). I work with Ruby, but I'm no expert. Do you have an example test that does such a thing. I looked at redis.rb, mongodb.rb, mysql.db, and I didn't see any such thing, but if you could point me to an example, I'd be happy to give it a go.